portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,11 +32,16 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  
-  private List<String> messages = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query("Comment");
```

Comment:
I'll suggest again inlining this into the usage site. It doesn't harm readability, and it removes one more thing that readers need to keep track of.

Reply:
The line highlighted is the query line. Did you mean combining the ones below it?

Reply:
(referring to this exact line, query. by "again" i just meant "i like to ask people to inline variables with one usage site")